python/cuml/neighbors/nearest_neighbors.pyx

```diff
@@ -43,7 +43,7 @@ cdef extern from "knn/knn.h" namespace "ML":
         int N
 
     cdef cppclass kNN:
-        kNN(int D) except +
+        kNN(int D, bool verbose) except +
```

Comment:
Same comment regarding verbose as the C++ level. Though we could use raise an issue to discuss whats the best approach to set verbosity.